api/VTTCue.json

```diff
@@ -413,10 +413,10 @@
               "version_added": false
             },
             "safari": {
-              "version_added": false
+              "version_added": "14.1"
```

Comment:
This is a little surprising since `VTTRegion` shipped much earlier, but seems correct, it's a recent fix by @frankolivier in https://trac.webkit.org/changeset/270738/webkit.